src/component.js

```diff
@@ -105,23 +125,23 @@ export function insertPptm(env : string = getEnv(), isDebug : boolean = getDebug
       }
     }
   } catch (err) {
-    logger.error('insertPPTM', err);
+    logger.error("insertPPTM", err);
   }
 }
 
 function listenForButtonRender() {
-  getEventEmitter().on('button_render', () => {
+  getEventEmitter().on("button_render", () => {
     window.paypalDDL = window.paypalDDL || [];
-    const buttonRenderEvent = window.paypalDDL.filter(e => e.event === 'paypalButtonRender');
+    const buttonRenderEvent = window.paypalDDL.filter(
+      (e) => e.event === "paypalButtonRender"
+    );
     if (buttonRenderEvent.length === 0) {
-      window.paypalDDL.push({ event: 'paypalButtonRender' });
+      window.paypalDDL.push({ event: "paypalButtonRender" });
     }
   });
 }
 
 export function setup() {
-  document.addEventListener('DOMContentLoaded', () => {
-    insertPptm(getEnv(), getDebug());
-  });
+  waitForWindowReady().then(() => insertPptm(getEnv(), getDebug()));
```

Comment:
I actually think we can remove this event listener entirely. It's adding a script to `<head>` -- the window does not need to be ready to do that.

Reply:
@bluepnume good point! We should have looked at what `insertPptm()` is doing. I agree that we don't need `waitForWindowReady` at all since it can fire right away: https://github.com/paypal/paypal-muse-components/blob/cc15563b658e258ae89b78cfb4a47c9bdb0585af/src/component.js#L104

Reply:
Good point! I'm headed back to the start.